ReactAndroid/src/main/java/com/facebook/react/uimanager/JSTouchDispatcher.java

```diff
@@ -72,6 +73,7 @@ public void handleTouchEvent(MotionEvent ev, EventDispatcher eventDispatcher) {
       // {@link #findTargetTagForTouch} to find react view ID that will be responsible for handling
       // this gesture
       mChildIsHandlingNativeGesture = false;
+      mGestureStartTime = ev.getEventTime();
```

Comment:
Can you please unset this to a default value (e.g. MIN_VALUE) or set/unset a new boolean when the touch ends? And then assert that wherever we use mGestureStartTime that it's actually been initialized properly/we're in the middle of a touch?